Fix summarization transform serialization and multi-agent wiring (#132)
oracle/wayflow: docs/wayflowcore/source/core/howtoguides/howto_managerworkers.rst

to the manager's rendered prompt in a multi-agent setup.
These component-level transforms are prepended to any transforms already present on the ``managerworkers_template``.
If ``group_manager`` is itself an ``Agent`` with ``Agent(..., transforms=[...])``, those manager-agent transforms are also preserved during execution.

.. literalinclude:: ../code_examples/howto_managerworkers.py
    :language: python
    :start-after: .. start-##_Managerworkers_pattern_with_transforms
    :end-before: .. end-##_Managerworkers_pattern_with_transforms

.. _managerworkers_transform_ordering:

.. note::
   **How ``transforms`` and ``managerworkers_template`` work together**

   During execution, WayFlow builds the manager agent's effective runtime template from the configured ``managerworkers_template``.
   If ``group_manager`` is an ``Agent`` with its own pre-rendering transforms, those run first, then the transforms passed through ``ManagerWorkers(..., transforms=[...])``, and finally any pre-rendering transforms already attached directly to the template.
   The effective order is therefore:

>    1. ``Agent(..., transforms=[...])`` for the group manager, when ``group_manager`` is an ``Agent``

Reviewer: ```suggestion
   1. ``group_manager.transforms`` for the group manager, when ``group_manager`` is an ``Agent``
```

Author: applied

>    2. ``ManagerWorkers.transforms`` in the order you passed them

Reviewer: ```suggestion
   2. ``manager_worker.transforms`` in the order you passed them
```

Author: applied

>    3. ``managerworkers_template.pre_rendering_transforms``

Reviewer: ```suggestion
   3. ``manager_worker.template.pre_rendering_transforms``
```

Author: manager_worker.template.pre_rendering_transforms doesn't exist, keeping the original one

>    4. Template rendering and manager execution

Reviewer: not needed?

Author: Modified
